### ISSUE TITLE
Update credits

### ISSUE THREAD

**Issue by terracoda** [phetsims/resistance-in-a-wire#208]
From https://github.com/phetsims/a11y-research/issues/134

The credits for this sim are missing a few people:
- Emily Moore and Taliesin Smith need to be added to the Team list
- Sound designers are  missing, too.

Reaching out to @Ashton-Morris and @emily-phet. Are Mike Winters and Ashton Morris the sound designers for this sim?

**Comment by terracoda** [comment]
Once we have all the names, please assign to @jbphet and mark as Priority 3.

**Comment by terracoda** [comment]
@emily-phet, could you verify the sound designers for Resistance in a Wire?

**Comment by emily-phet** [comment]
@Ashton-Morris and @mikewinters10

**Comment by samreid** [comment]
Assigned to responsible dev @zepumph to add `soundDesign: 'Ashton Morris, Mike Winters',` to the credits.

**Comment by terracoda** [comment]
@zepumph, the A11y Team still needs to be added, too.
**Current Screen Shot of Published Sim:**
<img width="716" alt="Screen Shot 2019-06-05 at 2 04 08 PM" src="https://user-images.githubusercontent.com/1902116/58954955-e9e01180-879a-11e9-9368-668e0a2cc064.png">

Please **add bolded names** to Team without bold of course ;-):
- Team: Wendy Adams, Mindy Gratny, **Emily Moore**, Ariel Paul, **Taliesin Smith**

Please add sound designers after QA:
- Sound Design: Ashton Morris, Mike Winters

**Comment by emily-phet** [comment]
Also - "Brianna Tomlinson" to end of team list.

**Comment by terracoda** [comment]
@emily-phet, good catch, I added @briannatomlinson to the list in https://github.com/phetsims/resistance-in-a-wire/issues/208#issuecomment-499057633, and will be updating issues for other sims.

**Comment by zepumph** [comment]
Implemented above, I also converted many usages of "Emily Moore" to "Emily B. Moore". Closing

**Comment by zepumph** [comment]
Reopening. I should publish a MR so that the proper credits are published.

**Comment by zepumph** [comment]
I should also add "Emily Moore" to bad text, in preference with `Emily B. Moore`

**Comment by terracoda** [comment]
@zepumph, this issue is a bit old, but it has not yet been done. Maybe it fell on your radar?
I updated the credits on the web site, adding Emily, myself, Brianna, and Ashton Morris and Mike Winters.

![Screen Shot 2020-10-09 at 1 32 18 PM](https://user-images.githubusercontent.com/1902116/95605716-e7976280-0a33-11eb-9738-c4d0463048a8.png)

**Comment by mikewinters10** [comment]
Could "Mike Winters" be included as well under sound?

**Comment by zepumph** [comment]
Ahh yes @terracoda, thanks for the ping. I just updated the credits on the 1.6 branch and published them to the website. Please review. 

https://phet.colorado.edu/sims/html/resistance-in-a-wire/latest/resistance-in-a-wire_en.html
